src/LiveComponent/src/EventListener/AddLiveAttributesSubscriber.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\UX\LiveComponent\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\UX\LiveComponent\Twig\LiveComponentRuntime;
+use Symfony\UX\TwigComponent\ComponentAttributes;
+use Symfony\UX\TwigComponent\EventListener\PreRenderEvent;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class AddLiveAttributesSubscriber implements EventSubscriberInterface
+{
+    public function __construct(private LiveComponentRuntime $runtime)
+    {
+        // todo make lazy?
+    }
+
+    public function onPreRender(PreRenderEvent $event): void
+    {
+        if (!isset($event->config['live'])) {
```

Comment:
```suggestion
        if (!i$event->config['live'] ?? false) {
```

Would allow for `live: false`, not that this is a use-case.